project-management/src/main/java/life/qbic/projectmanagement/domain/model/measurement/NGSMeasurement.java

```diff
@@ -74,33 +124,26 @@ private NGSMeasurement(ProjectId projectId, Collection<SampleId> sampleIds,
    * @param sampleIds  the sample ids of the samples the measurement was performed on. If more than
    *                   one sample id is provided, the measurement is considered to be performed on a
    *                   pooled sample
-   * @param instrument the instrument used for the measurement, which is represented as an
-   *                   {@link OntologyTerm}
    * @return
    * @since 1.0.0
    */
   public static NGSMeasurement create(ProjectId projectId, Collection<SampleId> sampleIds,
-      MeasurementCode measurementCode, OntologyTerm instrument) {
+      MeasurementCode measurementCode, Organisation organisation, NGSMethodMetadata method,
+      String comment) throws IllegalArgumentException {
     if (sampleIds.isEmpty()) {
       throw new IllegalArgumentException(
           "No sample ids provided. At least one sample id must provided for a measurement.");
     }
-    requireNonNull(instrument);
     requireNonNull(measurementCode);
+    requireNonNull(method);
```

Comment:
It would be great to have an error message here
```suggestion
    requireNonNull(method, "method must not be null");
```